tests/Webhooks/RegistryTest.php

```diff
@@ -408,6 +408,29 @@ public function testProcessWebhook()
         $this->assertNull($response->getErrorMessage());
     }
 
+    public function testProcessDomainEvent()
+    {
+        $handler = $this->getMockHandler();
+        $handler->expects($this->once())
+            ->method('handle')
+            ->with(
+                Topics::CUSTOMERS_SMS_MARKETING_SUBSCRIBED,
```

Comment:
Could we instead create a mock set of topics that aren't dependent on internal features?

```rb
final class TestTopics
{
  public const SOMETHING_MADE_UP = 'SOMETHING_MADE_UP';
}

// Example: invoke with topic `something.made_up`
```

Reply:
That should be fine as random strings are allowed to account for uncovered topics!